src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -37,20 +37,38 @@ import {
 
 let nextUniqueId = 0;
 
+/**
+ * The `sky-radio-group` component organizes radio buttons into a group. It is required for radio
+ * buttons on Angular reactive forms, and we recommend using it with all radio buttons.
+ * On Angular forms, the component manages the selected values and keeps the forms up-to-date.
+ * When users select a radio button, its value is driven through an `ngModel` attribute that you specify on the `sky-radio-group` element.
+ */
 @Component({
   selector: 'sky-radio-group',
   templateUrl: './radio-group.component.html'
 })
 export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit, OnDestroy {
 
+/**
+ * Sets the radio button group's `aria-labelledby` attribute to support
+ * [accessibility](https://developer.blackbaud.com/skyux/components/radio#accessibility). The value
+ * should be the HTML element ID (without the leading #) of the element that labels the radio
+ * button group. If the radio button group does not include a visible label on the screen, use the `ariaLabel` property instead.
+ */
   @Input()
   public ariaLabelledBy: string;
 
+/**
+ * Defines a string value to label the radio button group and set the `aria-label` attribute to
```

Comment:
This link is busted. I think we can just remove the link, but I made a note elsewhere about how to incorporate the *Accessibility* section content into property descriptions.

Reply:
The link is still busted, and I'd recommend incorporating some of the content from the *Accessibility* section into this description.

Reply:
Added Accessibility section but removed the link to it.